src/main/java/net/buildtheearth/buildteam/components/generator/field/menu/CropStageMenu.java

```diff
@@ -0,0 +1,179 @@
+package net.buildtheearth.buildteam.components.generator.field.menu;
+
+import net.buildtheearth.Main;
+import net.buildtheearth.buildteam.components.generator.Settings;
+import net.buildtheearth.buildteam.components.generator.field.Crop;
+import net.buildtheearth.buildteam.components.generator.field.CropStage;
+import net.buildtheearth.buildteam.components.generator.field.Field;
+import net.buildtheearth.buildteam.components.generator.field.FieldFlag;
+import net.buildtheearth.buildteam.components.generator.field.FieldSettings;
+import net.buildtheearth.utils.Item;
+import net.buildtheearth.utils.Liste;
+import net.buildtheearth.utils.menus.AbstractMenu;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.Sound;
+import org.bukkit.entity.Player;
+import org.bukkit.event.inventory.ClickType;
+import org.bukkit.inventory.ItemStack;
+import org.ipvp.canvas.mask.BinaryMask;
+import org.ipvp.canvas.mask.Mask;
+
+public class CropStageMenu extends AbstractMenu {
+
+    public static String CROP_TYPE_INV_NAME = "Choose a Crop Stage";
+
+
+    private final Crop crop;
+    private final byte STAGE_ONE_SLOT = 11;
+    private final byte STAGE_TWO_SLOT = 15;
+
+    public CropStageMenu(Player player, Crop crop) {
+        super(3, CROP_TYPE_INV_NAME, player);
+        player.sendMessage("TEST " + crop.toString());
+        this.crop = crop;
+    }
+
+
+    @Override
+    protected void setPreviewItems() {
+        Field field = Main.buildTeamTools.getGenerator().getField();
+        Crop crop = Crop.getByIdentifier(field.getPlayerSettings().get(getMenuPlayer().getUniqueId()).getValues().get(FieldFlag.CROP));
+
+        ItemStack itemOne = Item.create(Material.BARRIER);
+        ItemStack itemTwo = Item.create(Material.BARRIER);
+
+        switch (crop) {
+            case POTATO:
+                itemOne = Item.create(Material.STONE_PLATE, "§bLow", Liste.createList("", "§8Leftclick to select", "§8Rightclick for more information"));
+                itemTwo = Item.create(Material.STONE, "§bTall", Liste.createList("", "§8Leftclick to select", "§8Rightclick for more information"));
```

Comment:
Maybe use normal grass and tall grass to demonstrate that. Stone and Stone Plate looks a bit weird